backend/src/main/java/com/woowacourse/ternoko/service/ReservationService.java

```diff
@@ -0,0 +1,67 @@
+package com.woowacourse.ternoko.service;
+
+import com.woowacourse.ternoko.domain.FormItem;
+import com.woowacourse.ternoko.domain.Interview;
+import com.woowacourse.ternoko.domain.Location;
+import com.woowacourse.ternoko.domain.Member;
+import com.woowacourse.ternoko.domain.Reservation;
+import com.woowacourse.ternoko.dto.FormItemRequest;
+import com.woowacourse.ternoko.dto.ReservationRequest;
+import com.woowacourse.ternoko.repository.FormItemRepository;
+import com.woowacourse.ternoko.repository.InterviewRepository;
+import com.woowacourse.ternoko.repository.MemberRepository;
+import com.woowacourse.ternoko.repository.ReservationRepository;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class ReservationService {
+
+    private final MemberRepository memberRepository;
+    private final FormItemRepository formItemRepository;
+    private final ReservationRepository reservationRepository;
+    private final InterviewRepository interviewRepository;
+
+    public Long create(Long coachId, ReservationRequest reservationRequest) {
```

Comment:
팀 컨벤션인 `final`을 붙여줘야할 것 같아요

```suggestion
    public Long create(final Long coachId, final ReservationRequest reservationRequest) {
```